osu.Game/Overlays/Dashboard/Friends/FriendDisplay.cs

```diff
@@ -193,32 +163,33 @@ private List<User> getUsersInCurrentGroup()
             }
         }
 
-        private void addContentToPlaceholder(Drawable content)
+        private void addTableToPlaceholder(FillFlowContainer<UserPanel> table)
```

Comment:
This name had me quite confused for a second; if I'm understanding the intent correctly I'd rename to something like `replacePreviousContent`.